### PR TITLE
Maintenance mode

### PR DESCRIPTION
**Pull request by Jkrzy**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/706)

## What does this change?
1. Adds functionality to enable a maintenance mode via environment variable.
2. For GET requests to `/report/`, returns a 503 response and custom error page

~TODO~ Done
- [x] Create template per Acceptance Criteria

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Re-check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by Jkrzy** [comment]
Thanks for the initial template @hakhalid11!!! 

I made a few tweaks, removed the url path, added ES translations, and rebased against `develop`

This one is ready for review

**Comment by cantsin** [comment]
@Jkrzy we have a translation file conflict :point_down:

File: crt_portal/cts_forms/tests/test_views.py [hunk 1]
```diff
@@ -137,3 +137,19 @@ def test_only_keep_errors_from_submitted_form(self):
...
         self.assertEqual(response.context['contact_form'].initial['contact_email'], initial)
+
+
+class CRTReportWizardTests(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.url = reverse('crt_report_form')
+
+    @override_settings(MAINTENANCE_MODE=True)
```
Comment (cantsin) [inline crt_portal/cts_forms/tests/test_views.py:148]: Yessss. :raised_hands:

File: docs/maintenance_or_infrequent_tasks.md [hunk 2]
```diff
@@ -296,3 +296,35 @@ cf run-task crt-portal-django  -c "python crt_portal/manage.py refresh_trends" -
...
+cf restart crt-portal-django --strategy rolling
+```
+
+To **disable** maintenance mode and return the application to normal operations, remove the `MAINTENANCE_MODE` variable from the desired environment and restart all running instances.
+
+```shell
+cf target -s {target environment}
+cf unset-env crt-portal-django MAINTENANCE_MODE
+cf restart crt-portal-django --strategy rolling
+```
```
Comment (cantsin) [inline docs/maintenance_or_infrequent_tasks.md:323]: I think we should have a practice run with @ilodidoj on the above once we get this in. Love the documentation! :heart:

**Review by cantsin — CHANGES_REQUESTED** [review]
Other than the merge conflict, I would like to see the template tweaked a bit so that the icons aren't so lopsided:



Otherwise looks great :amaze:

**Comment by cantsin** [comment]
Update: I think I was on the wrong commit previously; looks great!

**Comment by Jkrzy** [comment]
Thanks @cantsin!! Rebased against develop to resolve the merge conflict in translation file.

I agree on the phone numbers being inconsistent amongst each other. This snippet matches how we're rendering them in the site footer, I'd love to tackle remediating if necessary in another card where we can settle on "how all our numbers should look"

**Review by cantsin — APPROVED** [review]
LGTM!

**Pull request merged**